### ISSUE TITLE
[ACCESSIBILITY] Find a place search button is missing form label

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#541]
On the explorer, the "find a place" search bar button is missing a form label. This occurs in the header and in the location panel search bar.

- What It Means: A form control does not have a corresponding label.
- Why It Matters: If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.
- What To Do: If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.

![image](https://user-images.githubusercontent.com/76966798/106819269-64c39480-662e-11eb-90b9-c0aa446a2ee3.png)
